docs/source/integrations/middleware.mdx

```diff
@@ -376,6 +376,21 @@ async function startApolloServer(typeDefs, resolvers) {
     typeDefs,
     resolvers,
     plugins: [ApolloServerPluginStopHapiServer({ hapiServer: app })],
+    // shared route options
```

Comment:
This doesn't look right — these parameters are to applyMiddleware, not the constructor, right?

Reply:
Correct